README.md

```diff
@@ -374,11 +374,13 @@ To run queries with the MarkLogic JavaScript and XQuery Debugger, a user will ne
 - **xdmp-eval-modules-change**: to use a non-default modules database or modules root
 - **xdmp-eval-modules-change-file**: to use the filesystem for modules
 
-For debugging, a user must also have at least one of these privileges:
+For debugging, a user must also have at least one of these privileges to evaluate a JavaScript or XQuery module in debug mode.
 
 - **debug-my-request**: for debugging requests launched by the debug user only
```

Comment:
Should be "requests" instead of "request", which also applies to the sentence below it.